opt/slackbutt/plugins/spacestation.py

```diff
@@ -0,0 +1,53 @@
+
+import re
+import slackbot.bot
+import requests
+from datetime import datetime, timedelta
+
+URL = 'http://api.open-notify.org/iss-pass.json?lat={0}&lon={1}'
+
+
+class SpaceStation():
+    def __init__(self):
+        self._latitude = 45.522005
+        self._longitude = 45.522005
+
+#    @property
```

Comment:
Obliged request release this property again and create setter so non-Portlandia users can ask for ISS fly-overs.